osu.Game.Tests/osu.Game.Tests.csproj

```diff
@@ -21,5 +21,6 @@
     <ProjectReference Include="..\osu.Game.Rulesets.Catch\osu.Game.Rulesets.Catch.csproj" />
     <ProjectReference Include="..\osu.Game.Rulesets.Mania\osu.Game.Rulesets.Mania.csproj" />
     <ProjectReference Include="..\osu.Game.Rulesets.Taiko\osu.Game.Rulesets.Taiko.csproj" />
+    <ProjectReference Include="..\osu.Desktop\osu.Desktop.csproj" />
```

Comment:
This can't really exist, which is why I was asking for the test to be provided in non-committed format. But it's fine to leave like this for now, we'll need to figure out the correct path forward here.